ext/intl/spoofchecker/spoofchecker.stub.php

```diff
@@ -64,4 +75,5 @@ public function setChecks(int $checks): void {}
     /** @tentative-return-type */
     public function setRestrictionLevel(int $level): void {}
 #endif
+    public function setAllowedChars(string $pattern, int $pattern_options = 0): void {}
```

Comment:
The parameter naming style should follow the extension's naming style, which is camelCase.

```suggestion
    public function setAllowedChars(string $pattern, int $patternOptions = 0): void {}
```